openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -284,13 +285,24 @@ def get(self, request):
     def list(self, request):
         """
         GET /api/user/v1/accounts?username={username1,username2}
+        GET /api/user/v1/accounts?email={user_email}
         """
         usernames = request.GET.get('username')
+        user_email = request.GET.get('email')
+        search_usernames = []
+
+        if usernames:
+            search_usernames = usernames.strip(',').split(',')
+        elif user_email:
+            try:
+                user_email = user_email.strip('')
+                user = User.objects.get(email=user_email)
+                search_usernames = [user.username]
```

Comment:
nit: you can pull the non `User.objects.get()` lines out to be before and after the try/except block.